torchao/prototype/custom_fp_utils.py

```diff
@@ -24,7 +24,9 @@ def _n_ones(n: int) -> int:
 F32_EXP_BIAS = _n_ones(EBITS_F32 - 1)
 
 
-def _f32_to_floatx_unpacked(x: Tensor, ebits: int, mbits: int) -> Tensor:
+def _f32_to_floatx_unpacked(
+    x: Tensor, ebits: int, mbits: int, fake_quantize: bool = False
```

Comment:
do all tests pass if you just make your behavior the default?  It's not ideal to have product-logic-specific branches like this.

Reply:
I removed the branches by adding a compute dtype instead, please take another look. I want to avoid changing numerics for existing use cases